### PR TITLE
docs: :memo: update readme and help in analyse.py

### PR DESCRIPTION
**Pull request by RBMindee**
Slight fixes and modification of the doc:

- fixed typo in readme suggested code using demonstrating synthesize
- Added the mention to tensorflow-addons
- Added model names in front of papers
- Mentionned in help of the noblock option of analyze.py that it's only used for CI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Or even rebuild the original document from its predictions:
 ```python
 import matplotlib.pyplot as plt
 
-plt.imshow(result.synthesize()); plt.axis('off'); plt.show()
+plt.imshow(result.synthesize()[0]); plt.axis('off'); plt.show()
```
Comment (fg-mindee) [inline README.md:72]: Was already fixed in #590 :sweat_smile: Mind reversing that change please? :pray:

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ For examples & further details about the export format, please refer to [this se
 
 ### Prerequisites
 
-Python 3.6 (or higher) and [pip](https://pip.pypa.io/en/stable/) are required to install docTR. Additionally, you will need to install at least one of [TensorFlow](https://www.tensorflow.org/install/) or [PyTorch](https://pytorch.org/get-started/locally/#start-locally).
+Python 3.6 (or higher) and [pip](https://pip.pypa.io/en/stable/) are required to install docTR. Additionally, you will need to install at least one of [TensorFlow](https://www.tensorflow.org/install/) or [PyTorch](https://pytorch.org/get-started/locally/#start-locally). If you use tensorflow, you will need to install the additional package `tensorflow-addons`
```
Comment (fg-mindee) [inline README.md:92]: What do you think about always specifying the extra with the pip install command?
Because here we say that TF needs `tensorflow-addons` but say tomorrow there are 5 extra TF-specific deps, we don't want to leave the user alone to do it :/ 

But if they run `pip install python-doctr[tf]` that will automatically install the remaining dependencies. What do you think?

**Review by fg-mindee — CHANGES_REQUESTED** [review]
thanks!
About the page synthesis instruction, that was fixed in #590! I added some comment about the tf-addons installation :)

File: README.md (hunk 2 above)
Comment (RBMindee) [reply README.md:93]: I find it a bit awkward that you have "you should install tensorflow first" and "you can then install the latest release with pip install python-doctr" while it does not produce a working installation. I understand that you propose a solution immediately after,  but doesn't it mean that python-doctr setup alone should always need an argument ?

Anyway, that's a very minor issue and the main correction was the error in the example, so I guess we can cancel this PR if you do not like this correction ^^"

Comment (fg-mindee) [reply README.md:93]: Yup I would argue it should, and thus we can remove the section about "you need to install a framework first" :+1: 

Oh don't take it the wrong way, I was thinking about trying to harmonize installation instructions. I'm only trying to find the best option to tackle the underlying installation issue here

Comment (RBMindee) [reply README.md:93]: No worries! I was just saying that messing with setup tool is probably a bit more than twiddling the doc, but we can keep it in the PR if you want. If I understood you correctly, setup_tools should throw an error if an argument is not specified ?

Comment (RBMindee) [reply README.md]: Actually, thinking a bit more, I guess it makes sense to have a framework agnostic version, even if it's not standalone. We could simply have a warning in the installation instruction. What do you think about the current state of things ?

Comment (fg-mindee) [reply README.md]: Yup I was only suggesting to modify the instructions in the README here :) 
We'll have to do the same with the instructions in the documentation but that can wait for another PR

File: README.md [hunk 3]
```diff
@@ -105,11 +105,13 @@ For Windows users, those dependencies are included in GTK. You can find the late
 
 You can then install the latest release of the package using [pypi](https://pypi.org/project/python-doctr/) as follows:
 
+> :warning: Notice that the basic installation is not standalone, as it does not provide a deep learning framework, which is required for the package to run.
+
```
Comment (fg-mindee) [inline README.md:109]: One last adjustment: could you rephrase "Notice" to "Please note" ? :pray: 
And move the warning right after the `pip install python-doctr` command?

**Review by fg-mindee — COMMENTED** [review]
Last adjustments and we're good!

**Review by fg-mindee — APPROVED** [review]
Thanks a lot for the edits!
